cmd/cli/version_test.go

```diff
@@ -170,6 +183,19 @@ func TestOutputVersionInfo(t *testing.T) {
 			expected:   "CLI Version: version.Info{Version:\"v0.0.0\", GitCommit:\"xxxxxxx\", BuildDate:\"0000-00-00-00:00\"}\nMesh [test] Version: version.Info{Version:\"v0.0.0\", GitCommit:\"xxxxxxx\", BuildDate:\"0000-00-00-00:00\"}\n",
 			clientOnly: false,
 		},
+		{
+			name: "cli and mesh versions with no remote version info",
+			versionInfo: versionInfo{
+				cliVersionInfo: &version.Info{
+					Version:   "v0.0.0",
+					GitCommit: "xxxxxxx",
+					BuildDate: "0000-00-00-00:00",
+				},
+				remoteVersionInfo: nil,
+			},
+			expected:   "CLI Version: version.Info{Version:\"v0.0.0\", GitCommit:\"xxxxxxx\", BuildDate:\"0000-00-00-00:00\"}\n",
+			clientOnly: false,
+		},
```

Comment:
Could we extend the unit test to cover the expected to be : `Mesh Version: No control plane found in namespace`